client/forest-manager/src/traits.rs

```diff
@@ -0,0 +1,24 @@
+use sp_core::H256;
+use sp_trie::CompactProof;
+use storage_hub_infra::types::{Key, Metadata, Proven};
+
+pub struct ForestProof {
```

Comment:
This seems reasonable now, although bare in mind it might change once we actually implement it, for convenience.